src/signals/incident/components/IncidentForm/services/custom-validators/index.js

```diff
@@ -0,0 +1,17 @@
+export function validateFileType(file, meta) {
+  if (meta && meta.allowedFileTypes && Array.isArray(meta.allowedFileTypes)) {
+    if (meta.allowedFileTypes.indexOf(file.type) === -1) {
+      return { custom: 'Dit bestand heeft niet het juiste type.' };
+    }
+  }
+  return null;
+}
+
+export function validateMaxFilesize(file, meta) {
+  if (meta && meta.maxFileSize) {
+    if (file.size >= meta.maxFileSize) {
+      return { custom: 'Dit bestand is te groot.' };
```

Comment:
It might be nice to also show the maximum file size to the user in case of this error